azure-sql/database/elastic-jobs-tsql-create-manage.md

```diff
@@ -54,28 +54,34 @@ In each of the target server(s)/database(s), create a contained user mapped to t
 - For example, to create a contained database login in the `master` database, and a user in the user database, based on the user-assigned managed identity (UMI) named `job-agent-UMI`:
 
 ```sql
---Create a login on the master database mapped to a user-assigned managed identity (UMI)
-CREATE LOGIN [job-agent-UMI] FROM EXTERNAL PROVIDER; 
+-- Connect to the master database of the Azure SQL logical instance of job agent
+-- Use universal with MFA authentication type
```

Comment:
Only Microsoft Entra auth is needed to create a login for EXTERNAL USER, any type will work. But, this is a good add.

```suggestion
-- To run these T-SQL scripts, use Microsoft Entra authentication for your database connection.
```